### PR TITLE
chore(python): show synchronous snippets

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: -

### Changes included:

showcase synchronous snippets, and mention the asynchronous class, for broader usage coverage https://algolia.slack.com/archives/C05C3DRTVFU/p1727267587674509

### CODE REVIEW

File: templates/python/snippets/method.mustache [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
+def snippet_for_{{#lambda.snakecase}}{{method}}{{/lambda.snakecase}}{{testIndex}}():
     """
     Snippet for the {{method}} method.
 
     {{{description}}}
     """
     # >SEPARATOR {{method}} {{testName}}
-    # Initialize the client
-    client = {{#lambda.pascalcase}}{{{client}}}{{/lambda.pascalcase}}("ALGOLIA_APPLICATION_ID", "ALGOLIA_API_KEY"{{#hasRegionalHost}}, "ALGOLIA_APPLICATION_REGION"{{/hasRegionalHost}})
+    # Initialize the client, in an asynchronous context, you can use {{#lambda.pascalcase}}{{{client}}}{{/lambda.pascalcase}} instead, which exposes the exact same methods.
```
Comment (millotp) [inline templates/python/snippets/method.mustache:16]: is this comment shown in the doc ?

Comment (shortcuts) [reply templates/python/snippets/method.mustache]: i'll see if this is enough https://github.com/algolia/api-clients-automation/pull/3832/commits/30dab08197a327f31a9e76f205df8c4322ccbd26

Comment (shortcuts) [reply templates/python/snippets/method.mustache]: all good now

Comment (shortcuts) [reply templates/python/snippets/method.mustache]: great catch gg

**Pull request merged**